deployments/stacks/dpe-k8s-deployments/main.tf

```diff
@@ -99,11 +102,15 @@ module "envoy-gateway" {
   git_revision         = var.git_revision
   namespace            = "envoy-gateway"
   argo_deployment_name = "envoy-gateway"
+  cluster_issuer_name  = "selfsigned"
+  # To determine more elegant ways to fill in these values, for example, if we have
+  # a pre-defined DNS name for the cluster (https://sagebionetworks.jira.com/browse/IT-3931)
+  ssl_hostname   = var.ssl_hostname
+  auth0_jwks_uri = var.auth0_jwks_uri

```

Comment:
Depends on https://sagebionetworks.jira.com/browse/IT-3931 to move away from selfsigned certs